lms/djangoapps/certificates/apis/v0/tests/test_views.py

```diff
@@ -102,6 +110,21 @@ def test_no_certificate(self):
             'no_certificate_for_user',
         )
 
+    def test_no_certificate_configuration(self):
+        self.cert.download_url = ''
```

Comment:
It would be good if you add a docstring for this test.

Reply:
Done.